src/test_topk.c

```diff
@@ -0,0 +1,152 @@
+#include <stdio.h> // printf
```

Comment:
Doesn't this file belong in the `tests` folder?

Reply:
Moved